web/src/pages/actions/[quickActionTaskUrlSlug].tsx

```diff
@@ -33,6 +33,12 @@ export const QuickActionElement = (props: Props): ReactElement => {
           <h1>{props.quickActionTask.name}</h1>
         </div>
       </div>
+      {props.quickActionTask.summaryDescriptionMd?.length > 0 && (
```

Comment:
dust, we don't need `?.length > 0` right? It would not render if there is an empty string.

Reply:
@FarazA22 We otherwise don't want to render the `<HorizontalRule />`, right?

Reply:
Yeah I think the HR would only render if the summary text renders.

Reply:
I think I agree with Faraz.